packages/apps-config/src/ui/logos/index.ts

```diff
@@ -248,6 +252,7 @@ export const namedLogos: Record<string, unknown> = {
   polkabtc: nodePolkaBTC,
   polkadex: nodePolkadex,
   polkadot: nodePolkadot,
+  polkafoundry: nodePolkaFoundry,
```

Comment:
You would just need an entry for `rococoPolkaFoundry: nodePolkaFoundry` as well (the `info` in the RPC endpoint matches on this name)

Reply:
@jacogr TY. I just added it.